### DISCUSSION TITLE
New Version of `config` Adapting Swin Transformer Algorithm: 新版 config 适配 Swin Transformer

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#338] [MMPreTrain]
# New Version of `config` Adapting Swin Transformer Algorithm: 
 新版 config 适配 Swin Transformer 

New Version of `config` Adapting Swin Transformer Algorithm:
 MMEngine adopts a revamped configuration system using which we intend to optimize the configuration files. 
 See [folder]([https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)](https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs))  for associated applications.
 Within this task, your goal is to carry out appropriate changes across all configuration files located inside the following location: 'configs/swin\_transformer/'.

新版 config 适配 swin-transformer算法 
 mmengine 实现了一套新的 config 机制，基于新的 config 机制进行配置文件的适配，[相关案例]([https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)](https://github.com/open-mmlab/mmpretrain/tree/main/mmpretrain/configs)) 
 该任务适配的 config 目录：configs/swin_transformer/ ，完成所有文件适配

## Task Information

**Repository**：MMPreTrain

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：timerring

**Claim Time**：2023年08月30日

**PR Link**：https://github.com/open-mmlab/mmpretrain/pull/1780

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by timerring** [comment]
https://github.com/open-mmlab/mmpretrain/pull/1780
